docs/05.PORT-API.md

```diff
@@ -151,6 +151,32 @@ jerry_port_get_native_module (jerry_value_t name) /**< module specifier */
 }
 ```
 
+## Promise
+
+```c
+/**
+ * HostPromiseRejectionTracker operations
+ */
+typedef enum
+{
+  JERRY_PROMISE_REJECTION_TRACKER_REJECT, /**< promise is rejected without any handlers */
+  JERRY_PROMISE_REJECTION_TRACKER_HANDLE, /**< handler is added to a rejected promise for the first time */
+} jerry_promise_rejection_tracker_t;
```

Comment:
I'm not sure about the names proposed here. This is definitely not the type of a promise rejection tracker (what would be intuitive from this name), but the type of operations. (The type of a tracker would be a function, I guess.)

How about
```c
typedef enum
 {
   JERRY_PROMISE_REJECTION_OPERATION_REJECT, /**< promise is rejected without any handlers */
   JERRY_PROMISE_REJECTION_OPERATION_HANDLE, /**< handler is added to a rejected promise for the first time */
 } jerry_promise_rejection_operation_t;
```
?

(Note: I've also considered suggesting `jerry_promise_rejection_tracker_operation`, as that would follow the pattern of other typedefs in the port API. But that would be too long for a name, I guess.)

Reply:
`jerry_promise_rejection_operation_t` will be fine.